docker-compose.override.local.yml

```diff
@@ -125,6 +125,13 @@ services:
       exit 0;
       "
 
+  qgis:
+    build:
+      target: dev
+    volumes:
+# Uncomment to use a local libqfieldsync dev environment
+#      - /home/user/dev/libqfieldsync:/libqfieldsync
+
```

Comment:
Can we use a envvar for this? I would prefer we never keep local modifications in the `docker-compose` files. So far all of these situations are handled with an envvar.

Reply:
The first part before the `:` needs to point to a valid directory. We could by default mount some random directory from the source tree. Or a dummy volume. Both approaches should work but seem clumsy, that's why I didn't implement one of them in the first place. Do you see an option I forgot?

Reply:
added